src/main/java/seedu/address/model/borrower/Email.java

```diff
@@ -23,9 +23,10 @@
     private static final String LOCAL_PART_REGEX = "^[\\w" + SPECIAL_CHARACTERS + "]+";
     private static final String DOMAIN_FIRST_CHARACTER_REGEX = "[^\\W_]"; // alphanumeric characters except underscore
     private static final String DOMAIN_MIDDLE_REGEX = "[a-zA-Z0-9.-]*"; // alphanumeric, period and hyphen
-    private static final String DOMAIN_LAST_CHARACTER_REGEX = "[^\\W_]$";
+    private static final String DOMAIN_LAST_CHARACTER_REGEX = "[^\\W_]{2,}$";
```

Comment:
Actually I think we should accept 1 character after dot also (unless our ug define valid email to have 2 or more characters after . ) Speaking of which, we need to update UG on what we accept ads valid email address

Reply:
Yup true.